src/GitTabExpansion.ps1

```diff
@@ -264,6 +276,23 @@ function GitTabExpansionInternal($lastBlock, $GitStatus = $null) {
 
     switch -regex ($lastBlock -replace "^$(Get-AliasPattern git) ","") {
 
+        # Handles git pr alias
```

Comment:
It's not a big perf hit but it strikes me that these "case" statements are less likely to be encountered than the basic `git <cmd>` completions.  As such, it seems like the new vsts-oriented "case" statements should go towards the bottom after the more common cases.

Reply:
Other than the above, this PR LGTM.